docs/rackhd/event_notification.rst

```diff
@@ -165,6 +165,40 @@ Example of heartbeat event payload:
         }
     }
 
+Example of node *discovered* event payload:
+
+.. code-block:: JSON
```

Comment:
The whole block data is not valid JSON data, for JSON data, each key must be quoted.